framework/src/main/java/org/tron/core/config/DefaultConfig.java

```diff
@@ -48,9 +47,7 @@ public RevokingDatabase revokingDatabase() {
     int dbVersion = Args.getInstance().getStorage().getDbVersion();
     RevokingDatabase revokingDatabase;
     try {
-      if (dbVersion == 1) {
-        revokingDatabase = RevokingStore.getInstance();
-      } else if (dbVersion == 2) {
+      if (dbVersion == 2) {
```

Comment:
same with above

Reply:
It will throw a runtime exception when dbVersion set to 1 as the else logic will take effect.